tests/Unit/Helpers/SessionYearTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Unit\Helpers;
+
+use App\Helpers\Models\LegiScan\SessionYear;
+use Tests\TestCase;
+use App\Models\LegiScan\Session;
+
+class SessionYearTest extends TestCase
+{
+    /** @test */
+    public function listOfYearsAreEqualToStartAndEndYearsInSessions()
+    {
+        Session::factory()->create([
+            'year_start' => 1,
+            'year_end' => 2,
+        ]);
+        Session::factory()->create([
+            'year_start' => 2,
+            'year_end' => 3,
+        ]);
+        Session::factory()->create([
+            'year_start' => 3,
+            'year_end' => 4,
+        ]);
+
+        $years = app(SessionYear::class)->getAll();
+
+        $this->assertTrue($years->duplicates()->isEmpty());
+
+        $this->assertTrue($years->diff([1, 2, 3, 4])->isEmpty());
```

Comment:
Just assert that the collection matches the array expectations

```
$this->assertEquals(
  [1, 2, 3, 4],
  $myCollection->toArray()
);
```

🤷 Way easier

Reply:
Done